source/scripts/runAndroidTests.sh

```diff
@@ -13,6 +13,33 @@ UNITY_PATH=$(awk -F= '/^editor/ {print $2}' $scriptsDir/unity.properties)
 RESULTS_PATH="artifacts/android-test-results.xml"
 LOG_PATH="artifacts/androidTestsRun.log"
 
+# Check for any Android emulator / device
+connected_devices=$(adb devices | grep -v "List" | grep "device$")
+if [ -z "$connected_devices" ]; then
+    echo "No Android device connected."
+    
+    # Search for device name
+    DEVICE=$(emulator -list-avds | head -n 1)
```

Comment:
On my side, i had to change this line to specify the full path to `emulator` command (same as line 25)